.github/workflows/ci.yml

```diff
@@ -174,15 +174,55 @@ jobs:
     - name: Second Run
       run: bin/phpunit --group twice
 
-  build_and_run_container:
-    name: Build and Run Container
+  build_and_run_containers:
+    name: Build, Run, Test Containers
     needs: code_style_security
     runs-on: ubuntu-latest
+    env:
+      ILIOS_DATABASE_URL: mysql://root:root@127.0.0.1:3306/ilios?serverVersion=5.7
```

Comment:
As long as we hope to take advantage of serverless Aurora, the `serverVersion` should be `5.6`, until further notice...

Reply:
Fixed here, and I went ahead and changed it everywhere as it is our minimum version.